config/params/testnet_e2e_config.go

```diff
@@ -24,6 +24,7 @@ func E2ETestConfig() *BeaconChainConfig {
 	e2eConfig.GenesisDelay = 10 // 10 seconds so E2E has enough time to process deposits and get started.
 	e2eConfig.ChurnLimitQuotient = 65536
 	e2eConfig.MaxValidatorsPerWithdrawalsSweep = 128
+	e2eConfig.MinPerEpochChurnLimitElectra = 256000000000
```

Comment:
I thought this would inherit from mainnet configs

Reply:
Nope, mainnet and minimal have different values for the churn limit.